src/rendering/Helpers.js

```diff
@@ -1,3 +1,10 @@
 export const deadEnemyXPosition = -9999
 export const rocketDefaultX = 80
 export const rocketDefaultY = 70
+
+export function bossPopup() {
+  let popup = document.createElement('div')
+  popup.id = 'bossPopup'
+  popup.innerHTML = 'THE BOSS IS COMING!!!'
+  document.querySelector('#game-background').append(popup)
```

Comment:
Please append `popup` to the `body`.

It doesn't make much sense to append popup to the game background I think...

With parallax our game background is moving, isn't it?